Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.yml

```diff
@@ -3967,8 +3967,8 @@ script:
       name: asset_id
       required: true
     name: core-get-asset-details
-    deprecated:marketplacev2: false
     deprecated: true
+    hidden: true
```

Comment:
@nbensalm-palo Do we have the command in the README as well? If so, need to remove it entirely from there also